src/muz/spacer/spacer_sat_answer.cpp

```diff
@@ -4,6 +4,7 @@
 #include "ast/scoped_proof.h"
 #include "smt/smt_solver.h"
 
+#include "tactic/portfolio/smt_strategic_solver.h"
```

Comment:
the python build is fails because it needs to know that spacer depends on tactic/portfolio

Reply:
will fix. forgot about python build

Reply:
oops, running into a cycle:  portfolio -> spacer -> fp -> portfolio

suggestions?